ts/nni_manager/training_service/reusable/aml/amlClient.ts

```diff
@@ -135,11 +135,19 @@ export class AMLClient {
     
     // Monitor error information in aml python shell client
     private monitorError(pythonShellClient: PythonShell, deferred: Deferred<any>): void {
+        pythonShellClient.on('stderr', function (error: any) {
+            // FIXME: The error will only appear in console.
+            // Still need to find a way to put them into logs.
+            console.error(`Python process stderr: ${error}`);
+        });
         pythonShellClient.on('error', function (error: any) {
-            deferred.reject(error);
+            console.error(`Python process fires error: ${error}`);
+            deferred.reject(error || new Error('AML client Python process unknown error.'));
         });
         pythonShellClient.on('close', function (error: any) {
```

Comment:
`on(event: "close", listener: () => void): this;`
https://github.com/extrabacon/python-shell/blob/master/index.ts#L449

This event has no argument at all.